src/swish/app.ms

```diff
@@ -20,13 +20,157 @@
 ;;; OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
 ;;; DEALINGS IN THE SOFTWARE.
 
+(include "swish/script-testing.ss")
```

Comment:
Why this include since it will be imported a few lines down?

Reply:
Perhaps Scheme won't find the library unless it's included?

Reply:
Probably worth a comment explaining that.

Reply:
I removed the `include` by setting `library-directories` so that we find the testing library.